pkg/windows/windows.go

```diff
@@ -970,7 +970,7 @@ func (vm *windows) ensureHNSNetworksAreRemoved() error {
 		err = wait.PollImmediate(retry.Interval, retry.Timeout, func() (bool, error) {
 			// reinitialize and retry on failure to avoid connection reset SSH errors
 			if err := vm.removeHNSNetwork(network); err != nil {
-				vm.log.V(1).Error(err, "error removing %s HNS network", "network", network)
+				vm.log.V(1).Info("error removing HNS network", "network", network, "err", err.Error())
```

Comment:
> A failure to remove an HNS network is a recoverable error,

True, and should also be captured in the error message.

Reply:
Lowering the verbosity of the message to debug will force the users to restart the operator in debug mode, to see the details of the error and then investigate.

Reply:
This is an expected outcome, so I don't see there to be a reason for investigation.
The reason we retry is because removing the HNS network will cause the network to go down.

The intention was for this to only be logged in debug logging, thus the existing `V(1)`.
Printing errors is alarming for users.